frontend/src/validation.js

```diff
@@ -0,0 +1,38 @@
+class Validation {
+    validateEmail(email) {
+        var errorMessage = "";
+        if(!email){
+            errorMessage = "Email is requered!"
+        }else{
+            const expression =/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/
```

Comment:
Extract regex expression into `regex.js` that contains expressions like:
`export const EMAIL_REGEX = /^(([^<>()[\]\...`